Makefile

```diff
@@ -23,3 +26,15 @@ typecheck:
 clean:
 	git clean -n dist
 	rm -v dist/**/*.{js,html}
+
+package:
+	cd dist/ && zip -r ../thankful.zip *
+
+publish:
+	# Use these to automatically publish
+	# https://www.npmjs.com/package/chrome-webstore-upload-cli
+	# https://www.npmjs.com/package/firefox-extension-deploy
+	node scripts/publish-mozilla-addons.sh
+	# Doing it like this would expose keys, not acceptable
+	#    webstore upload --source extension.zip --extension-id ${WEBSTORE_EXTENSION_ID} --client-id ${WEBSTORE_CLIENT_ID} --client-secret ${WEBSTORE_CLIENT_SECRET} --refresh-token ${REFRESH_TOKEN}
```

Comment:
`Webstore` by default gets these from envvars, you don't have to specify it.

And shouldn't we be safe by putting them as secret envvars in travis?

Reply:
Yeah, the only issue is that we'd want to name them something different in the envvars as they might otherwise conflict.